transports/azure-service-bus/queue-permissions.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Endpoint Managed Entity Queue Permissions
+summary: Explains queue-scoped permissions needed by an endpoint running in Azure Service Bus
+component: ASBS
+reviewed: 2025-10-14
+---
+
+When using managed entities queue-scoped permissions for endpoint connections to Azure Service Bus the following minimum permissions are required:
```

Comment:
Would it be worthwhile to mention that the transport is designed to work without manage rights as a general statement in the documentation?